cfme/utils/log_validator.py

```diff
@@ -1,17 +1,26 @@
 import re
 
-import pytest
-from _pytest.outcomes import Failed
-
 from .ssh import SSHTail
 from cfme.utils.log import logger
 from cfme.utils.wait import wait_for
 
 
+class LogValidatorException(Exception):
```

Comment:
I'd ask you to follow pep 8 conventions in naming exceptions. 
https://www.python.org/dev/peps/pep-0008/#exception-names

Reply:
@izapolsk LogValidator not raising actual error. If `fail patterns` matched rather than fail test with `pytest`; we are raising custom exception. `Error` at end of Exception point actual error occur in code as per me. Let me know view on it.

Reply:
actually this is not stopper but I'd rather use something like PatternNotMatchError.